src/Vgrep/Environment/Config/Sources/File.hs

```diff
@@ -322,3 +303,44 @@ styleToVty = \case
     Blink        -> Vty.blink
     Dim          -> Vty.dim
     Bold         -> Vty.bold
+
+
+instance FromJSON KeybindingsMonoid where
+    parseJSON = genericParseJSON jsonOptions
+
+instance FromJSON Command where
+    parseJSON = genericParseJSON jsonOptions
+
+instance FromJSON (Map Key.Chord Command) where
+    parseJSON = parseJSON >=> mapMKeys parseChord
+
+mapMKeys :: (Monad m, Ord k') => (k -> m k') -> Map k v -> m (Map k' v)
+mapMKeys f = fmap M.fromList . M.foldrWithKey go (pure [])
+  where
+    go k x mxs = do
+        k' <- f k
+        xs <- mxs
+        pure ((k', x) : xs)
```

Comment:
Depending on GHC 8.0 would break the build on Debian, so I can't use `-XApplicativeDo`.